library/GN.md

```diff
@@ -9,10 +10,32 @@ become the build system used on all platforms.
 
 In addition to the normal dependencies, you will need to install:
 * [ninja](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
-* [gn](https://gn.googlesource.com/gn/) 
+* [gn](https://gn.googlesource.com/gn/)
 
 Ensure that both binaries are in your path.
 
+Windows also requires the 64 bit compiler, linker and setup scripts to be in
```

Comment:
Let's add a
```
### Windows
```
before this section so it's easy for people to skip things that don't apply to their platform.